site/en/blog/deps-rems-99/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Deprecations and removals in Chrome 99
+description: >
+  A round up of the deprecations and removals in Chrome 99 to help you plan.
+layout: 'layouts/blog-post.njk'
+date: 2022-02-03
+updated: 2022-02-03
+hero: 'image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/km2rEb4m5xmqo3Y2KVMn.png'
+alt: >
+  Deprecations and Removals hero logo
+tags:
+  - deprecations
+  - removals
+  - chrome-99
+---
+
+{% include 'partials/see-all-dep-rem.md' %}
+
+Chrome 99 beta was released on February 3, 2022 and is expected to become the
+stable version in early March, 2022.
+
+## Remove Battery Status API on Insecure Origins
+
+[Battery Status API is no longer supported on insecure origins](https://chromestatus.com/feature/4878376799043584),
+such as HTTP pages or HTTPS iframes embedded in HTTP pages. The Battery Status
+API allows web developers to access, among other things, a system's battery
+charging level and whether it is being charged. It is a powerful feature that
+has been around for over a decade and, as such, was originally designed with
+different security constraints.
+
+## Remove font-family -webkit-standard
```

Comment:
Should this instance of "-webkit-standard" be surrounded by backticks? I'm not sure if that's the case, but it would match the way we treat the name in the next paragraph.

Reply:
Alternatively, you could say "Remove the WebKit standard font-family."

Reply:
It should if dcc supports code font in subheadings. I'm not sure it does so I didn't edit this.

Reply:
Not in a heading.